ninja.sh

```diff
@@ -0,0 +1,36 @@
+package: ninja
+version: "1.8.2"
+tag: "v1.8.2"
+source: https://github.com/ninja-build/ninja
+build_requires:
+ - "GCC-Toolchain:(?!osx)"
+prefer_system: .*
+prefer_system_check: |
+  which ninja && case `ninja --version | sed -e 's/.* //' | cut -d. -f1,2,3 | head -n1` in "^(0|2)\.|^1\.([0-7]|[9-99])\.\d*|^1\.8\.[0-1]" ) exit 1 ;; esac
+---
+#!/bin/bash -e
+
+echo "Building ALICE Ninja. To avoid this install Ninja >= 1.8.2"
```

Comment:
I'd drop this line. I know it's in other places, however if we override versions using defaults, this line won't change.

Reply:
ok